niworkflows/interfaces/bids.py

```diff
@@ -163,8 +163,7 @@ def _run_interface(self, runtime):
         self.layout = _init_layout(self.inputs.in_file,
                                    self.layout,
                                    self.inputs.bids_validate)
-        params = self.layout.parse_file_entities(self.inputs.in_file,
-                                                 domains=['bids'])
```

Comment:
I'm dropping the `domains` thing altogether. I thought it could be replaced with `scope`, but either that's not true or there's a bug in pybids. Either way, this doesn't seem to cause tests to fail, so...